node_modules/gh-users/lib/rest.js

```diff
@@ -340,18 +340,34 @@ GrassHopper.appRouter.on('post', '/api/users', function(req, res) {
  *
  * Import users using a CSV file
  *
+ * The CSV file should be formatted in the following way:
+ *
+ *  `displayName, email, shibbolethId`
+ *
+ * When importing a set of users using the local authentication strategy, the CSV format should be the following:
+ *
+ *  `displayName, email, password`
+ *
+ * Keep in mind that when this request finishes, the actual import is still going on. Depending on the size
+ * of your CSV file this might take a couple of minutes to complete
+ *
  * @Server      admin
  * @Method      POST
  * @Path        /users/import
  * @FormParam   {string}            authenticationStrategy  The authentication strategy for the user    [local,shibboleth]
  * @FormParam   {File}              file                    The CSV file to import
- * @FormParam   {number}            tenant                  The id of the tenant to which the users should be imported
- * @FormParam   {number[]}          [app]                   The id(s) of the app(s) to which the users should be associated
+ * @FormParam   {number}            [app]                   The id of the application into which the users should be imported. Defaults to the current application
```

Comment:
Do we use `app` as the parameter for the app id in lots of other places? If not, I'd suggest making this `appId` if possible.